python/src/trezorlib/btc.py

```diff
@@ -261,8 +269,8 @@ def copy_tx_meta(tx: messages.TransactionType) -> messages.TransactionType:
         return tx_copy
 
     this_tx = messages.TransactionType(
-        inputs=inputs,
-        outputs=outputs,
+        inputs=list(inputs),
```

Comment:
could we generate `messages.py` so that the arguments are `Sequence` instead of `List` ? then we wouldn't need this and presumably some other spots.

Reply:
Done in 28f32e0. Then I could not found any more places where we could get rid of `list()` transition - all the changes are in 4365425. On the contrary, I had to add a `list()` into one place where we are processing protobuf message (when I think about it now, I could have probably just change the return signature to contain `Sequence` - will do that in next commit)

Reply:
The above-mentioned done in 2306fddff. I hope the `_format_access_list()` is still working as expected, I could not resist modifying it to look better. However, it is only being used in a CLI function `cli/ethereum.px:sign_tx`, so there is no other way to test it than to have a transaction and let it signed via CLI. Or is there another non-manual way of testing these CLI functionalities?